app/views/shared/_navbar.html.slim

```diff
@@ -1,27 +1,30 @@
-.navbar.is-success[role="navigation" aria-label="main navigation"]
-  .navbar-brand
-    .navbar-item
+.navbar.has-background-light[role="navigation" aria-label="main navigation"]
+  .navbar-brand.left
+    .navbar-item.image.is-64x64
       = link_to image_tag("asana_logo.png", class: "asana_logo"), root_path
-    a.navbar-burger.burger[role="button" aria-label="menu" aria-expanded="false" data-target="navbarBasicExample"]
+    a.navbar-burger.burger[role="button" aria-label="menu" aria-expanded="false" data-target="navMenu"]
       span[aria-hidden="true"]
       span[aria-hidden="true"]
       span[aria-hidden="true"]
-  #navbar_test.navbar-menu
+  .navbar-menu#navMenu
     .navbar-start
       .navbar-item IF-089.Ruby 
       - if user_signed_in?
         a.navbar-item[href="/account/dashboard"]
           | Dashboard
         a.navbar-item[href="/account/projects"]
           | My projects
-      / .navbar-item.has-dropdown.is-hoverable
-        a.navbar-link
-          |  My Tasks 
-        .navbar-dropdown
-          a.navbar-item
-            |  List 
-          a.navbar-item
-            |  Calendar 
+        / .navbar-item.has-dropdown.is-hoverable
```

Comment:
it's OK to uncomment it and keep, just add href='#'

Reply:
done